src/trigger.c

```diff
@@ -0,0 +1,292 @@
+#include "common.h"
+
+void default_trigger_delegate(s32* arg0) {
+    arg0[0] |= 2;
+}
+
+void clear_trigger_data(void) {
+    CollisionStatus* collisionStatus = &gCollisionStatus;
+    s32 i;
+
+    if (!gGameStatusPtr->isBattle) {
+        gCurrentTriggerListPtr = &gTriggerList1;
+    } else {
+        gCurrentTriggerListPtr = &gTriggerList2;
```

Comment:
Just noticing these now but we can definitely name gTriggerList1 and gTriggerList2 bTriggerList and wTriggerList or something like that respectively